app/views/welcome/dashboard.html.erb

```diff
@@ -1,3 +1,2 @@
-
-<%= link_to 'Create a group' %>
-<%= link_to 'Join a group' %>
\ No newline at end of file
+<%= link_to 'Create a group', new_group_path %>
+<%= link_to 'Join a group' %>
```

Comment:
There is no link here?

Reply:
we haven't made a story/feature for joining a group yet, the story for the dashboard just says displays "join group" button, is that okay?

Reply:
It's not a blocker, but the problem is that you are testing for it and getting a false positive. The prudent thing would be to remove it (and the test), then add both when you are making the join group feature.